src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -449,9 +449,12 @@
     <value>In other binary operators:  &amp;&amp;   ||   ??   and   or</value>
     <comment>'and' and 'or' are C# keywords and should not be localized</comment>
   </data>
-  <data name="Show_name_suggestions" xml:space="preserve">
+  <data name="Show_name_s_uggestions" xml:space="preserve">
```

Comment:
https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/38111/Guide-for-setting-owners?anchor=titles
See the recommended title here, accelerator key is not needed in unified settings. But in our own legacy page we need that `_`
Many string changes in this PR is caused by this